cmd/runtimecfg/node-ip.go

```diff
@@ -252,7 +259,10 @@ func getSuitableIPs(retry bool, vips []net.IP, preferIPv6 bool, networkType stri
 		}
 
 		log.Errorf("Failed to find a suitable node IP")
-		time.Sleep(time.Second)
+		time.Sleep(time.Second * time.Duration(timerLoop))
+		if timerLoop == maxSecondsToSuitableIPsLoop {
+			timerLoop = 1
```

Comment:
We don't want to reset to 1 here, we want to cap it at 300. Once we hit a 5 minute delay we want to keep delaying 5 minutes until it succeeds. That's the key part of reducing the log spam.

Reply:
gotcha